helix-loader/src/lib.rs

```diff
@@ -263,11 +263,13 @@ mod merge_toml_tests {
         indent = { tab-width = 4, unit = "    ", test = "aaa" }
         "#;
 
-        let base: Value = toml::from_slice(include_bytes!("../../languages.toml"))
-            .expect("Couldn't parse built-in languages config");
+        // NOTE: Exact duplicate of helix_core::LanguageConfigurations::default()
+        let default: Value =
+            toml::from_str(&std::fs::read_to_string(crate::repo_paths::default_lang_configs()).unwrap())
```

Comment:
any case where unwrap will panic?

Reply:
If the repo languages.toml is moved from root without updating the path in helix_loader::repo_paths!